modules/apps/29-fee/keeper/keeper.go

```diff
@@ -168,28 +187,39 @@ func (k Keeper) GetAllFeeEnabledChannels(ctx sdk.Context) []types.FeeEnabledChan
 }
 
 // GetPayeeAddress retrieves the fee payee address stored in state given the provided channel identifier and relayer address
-func (k Keeper) GetPayeeAddress(ctx sdk.Context, relayerAddr, channelID string) (string, bool) {
-	store := ctx.KVStore(k.storeKey)
+func (k Keeper) GetPayeeAddress(ctx context.Context, relayerAddr, channelID string) (string, bool) {
+	store := k.storeService.OpenKVStore(ctx)
 	key := types.KeyPayee(relayerAddr, channelID)
 
-	if !store.Has(key) {
+	has, err := store.Has(key)
+	if err != nil {
+		panic(err)
+	}
+	if !has {
 		return "", false
 	}
 
-	return string(store.Get(key)), true
+	bz, err := store.Get(key)
+	if err != nil {
+		panic(err)
+	}
+
+	return string(bz), true
```

Comment:
Assuming these could be refactored to a single `Get` call and if `len(bz) == 0` then return `false` for found bool. 

Think this would save gas.

Reply:
Addressed in: https://github.com/cosmos/ibc-go/pull/7058/commits/3cd2a56eec88630bb058d3ec2d424105215fa7ee